cmd/fyne/commands/install.go

```diff
@@ -2,19 +2,75 @@ package commands
 
 import (
 	"errors"
-	"flag"
-	"fmt"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
 
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/cmd/fyne/internal/mobile"
+	"github.com/urfave/cli/v2"
 )
 
-// Declare conformity to Command interface
-var _ Command = (*installer)(nil)
+// Install returns the cli command for installing fyne applications
+func Install() *cli.Command {
```

Comment:
This sounds like what used to be `fyne package`? Except that it also Installs....
Humm... Now I don't know what to think about `Get` v.s. `Install`

Reply:
Outside of the scope of this PR. This has not changed since before.